CONTRIBUTING.md

```diff
@@ -148,8 +148,6 @@ poetry run bash scripts/format.sh
 poetry run bash scripts/run-ci-checks.sh
 ```
 
-If the spell checker catches errors, you will have to add the words to `.pyspelling-ignore-words`. (Note that if you wish to run the spellchecks manually on your local machine, first install `aspell` (using `brew` or `apt-get` or whatever package manager Windows uses).)
```

Comment:
Is this not relevant anymore?

Reply:
Good point / catch. I think when I removed that, I had thought that the spellchecks were covered there as well. Adding it back in now.